bip-0373.mediawiki

```diff
@@ -52,9 +52,11 @@ The new per-input types are defined as follows:
 | rowspan="2"| 0, 2
 |-
 | The MuSig2 aggregate public key (compressed) <ref>'''Why the compressed aggregate public key instead of x-only?'''
-BIP 32 requires public keys to include their evenness byte. Aggregate public keys are expected to be
-derived from, following [[bip-0328.mediawiki|BIP 328]], and therefore will
-need to include the evenness. Furthermore, PSBT_IN_TAP_BIP32_DERIVATION fields include fingerprints
+BIP 32 public keys can be derived from a BIP 327 MuSig2 aggregate public key (see: [[bip-0328.mediawiki|BIP 328]]).
+But since BIP 32 requires public keys to include their evenness byte, BIP 327 MuSig2 aggregate public keys must
+include their evenness byte as well.
+
+Furthermore, PSBT_IN_TAP_BIP32_DERIVATION fields include fingerprints
 to identify master keys, and these fingerprints require y-coordinate of the public key,
```

Comment:
While here can do this as well https://github.com/bitcoin/bips/pull/1705#discussion_r1874516388

```suggestion
to identify master keys, and these fingerprints require the y-coordinate of the public key,
```

Reply:
Missing "the" added.